functions/index.js

```diff
@@ -32,31 +32,58 @@ app.get("/workshops", function(request, response) {
     name: "workshops"
   });
 });
-app.get("/stempower19", function(request, response) {
-  response.render("pages/stempower19", {
+
+app.get("/stempower", function(request, response) {
+  response.render("pages/stempower", {
     name: "STEMpower"
   });
 });
+
+app.get("/stempower17", function(request, response) {
+  response.render("pages/stempower17", {
+    name: "STEMpower17"
+  });
+});
+
 app.get("/stempower18", function(request, response) {
   response.render("pages/stempower18", {
-    name: "STEMpower"
+    name: "STEMpower 2018"
   });
 });
 
-app.get("/stempower17", function(request, response) {
-  response.render("pages/stempower17", {
-    name: "STEMpower"
+app.get("/stempower19", function(request, response) {
+  response.render("pages/stempower19", {
+    name: "STEMpower 2019"
+  });
+});
+
+app.get("/fuse", function(request, response) {
+  response.render("pages/fuse", {
+    name: "fuse"
   });
 });
 
 app.get("/fuse17", function(request, response) {
   response.render("pages/fuse17", {
-    name: "fuse"
+    name: "fuse17"
   });
 });
+
 app.get("/fuse19", function(request, response) {
   response.render("pages/fuse19", {
-    name: "fuse"
+    name: "fuse 2019"
```

Comment:
Why is there a space here? I think the convention has been no spaces.

Reply:
wdym by convention?

Reply:
Convention meaning what does the existing code do

Reply:
IDK why there is a space. How does that impact the code though?

Reply:
It is better to use the same conventions throughout the code to reduce confusion

```suggestion
    name: "fuse19"
```

Reply:
Okie, I will change that